### PR TITLE
corrections coquilles

### PR DESCRIPTION
**Pull request by tetsuhaut**

### CODE REVIEW

File: files/fr/learn/css/building_blocks/sizing_items_in_css/index.html [hunk 1]
```diff
@@ -31,19 +31,19 @@ <h2 id="La_taille_naturelle_ou_intrinsèque_des_choses">La taille naturelle ou i
 
 <p>Tous les éléments HTML ont une taille "naturelle", définie avant toute modification par CSS. Un exemple parlant est celui d'un élément image. Une image a une largeur et une hauteur définies dans le fichier image qu'elle incorpore dans la page. On parle d'une <strong>taille intrinsèque </strong>— qui provient de l'image elle même.</p>
 
-<p>Si vous placer une image dans une page sans modifier sa hauteur ni sa largeur (que ce soit à en utilisant un attribut sur la balise <code>&lt;img&gt;</code> ou avec CSS), cela l'affichera en utilisant sa taille intrasèque. Ci-dessous nous avons l'exemple d'une image à laquelle nous avons ajouté une bordure afin de bien délimiter sa taille.</p>
+<p>Si vous placer une image dans une page sans modifier sa hauteur ni sa largeur (que ce soit à en utilisant un attribut sur la balise <code>&lt;img&gt;</code> ou avec CSS), cela l'affichera en utilisant sa taille intrinsèque. Ci-dessous nous avons l'exemple d'une image à laquelle nous avons ajouté une bordure afin de bien délimiter sa taille.</p>
```
Comment (audrasjb) [inline files/fr/learn/css/building_blocks/sizing_items_in_css/index.html:34]: Au passage :
"Si vous placer" > "Si vous placez"

**Review by audrasjb — CHANGES_REQUESTED** [review]
Hi, thanks for the PR!
The PR looks almost good to go, except we could fix another typo at the same time :)

File: files/fr/learn/css/building_blocks/sizing_items_in_css/index.html (hunk 1 above)
Comment (tetsuhaut) [reply files/fr/learn/css/building_blocks/sizing_items_in_css/index.html]: fait

Comment (audrasjb) [reply files/fr/learn/css/building_blocks/sizing_items_in_css/index.html]: 👍

File: files/fr/learn/css/building_blocks/sizing_items_in_css/index.html [hunk 2]
```diff
@@ -31,19 +31,19 @@ <h2 id="La_taille_naturelle_ou_intrinsèque_des_choses">La taille naturelle ou i
...
 
 <p>{{EmbedGHLiveSample("css-examples/learn/sizing/intrinsic-text.html", '100%', 500)}}</p>
 
-<p>Dans l'exemple ci-dessus; essayez d'ajouter du texte a l'interieur de l'élément vide. Les bordures contiennent maintenant ce texte car la hauteur de l'élément est définie par son contenu. De plus, la taille de cette <code>&lt;div&gt;</code> dans l'axe de bloc provient maintenant de la taille du contenu. Là aussi il s'agit de la taille intrasèque de l'élément — sa taille est définie par son contenu.</p>
+<p>Dans l'exemple ci-dessus; essayez d'ajouter du texte a l'interieur de l'élément vide. Les bordures contiennent maintenant ce texte car la hauteur de l'élément est définie par son contenu. De plus, la taille de cette <code>&lt;div&gt;</code> dans l'axe de bloc provient maintenant de la taille du contenu. Là aussi il s'agit de la taille intrinsèque de l'élément — sa taille est définie par son contenu.</p>
```
Comment (audrasjb) [inline files/fr/learn/css/building_blocks/sizing_items_in_css/index.html:42]: au passage et en attendant que la PR soit fusionnée :) 
"interieur" > "intérieur"

**Review by audrasjb — CHANGES_REQUESTED** [review]

**Review by tristantheb — CHANGES_REQUESTED** [review]
Missing translation EOP - Added on to-do

**Review by tristantheb — COMMENTED** [review]
Updated, wait for Sphinx proofreading before validating this PR

**Review by SphinxKnight — APPROVED** [review]
One last sentence needed, now we're good to go

**Pull request merged**